cloudinit/sources/DataSourceHetzner.py

```diff
@@ -3,15 +3,18 @@
 #
 # This file is part of cloud-init. See LICENSE file for license information.
 #
-"""Hetzner Cloud API Documentation.
-   https://docs.hetzner.cloud/"""
+# Hetzner Cloud API:
+# https://docs.hetzner.cloud/
```

Comment:
this change seems unrelated, and changes a docstring to a comment.
On master:

```
>>> from cloudinit.sources import DataSourceHetzner
>>> print(DataSourceHetzner.__doc__)
Hetzner Cloud API Documentation.
   https://docs.hetzner.cloud/
```

But on your branch:
```
>>> from cloudinit.sources import DataSourceHetzner
>>> print(DataSourceHetzner.__doc__)
None
```